pykafka/simpleconsumer.py

```diff
@@ -64,6 +64,7 @@ def __init__(self,
                  auto_commit_interval_ms=60 * 1000,
                  queued_max_messages=2000,
                  fetch_min_bytes=1,
+                 fetch_error_backoff_ms=500,
```

Comment:
I believe all of the changes to this file are unnecessary, since the `SimpleConsumer` never uses `fetch_error_backoff_ms`. It's ok for that kwarg to exist only for `RdKafkaSimpleConsumer`.